contents/docs/product-analytics/trends/statistical-analysis.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Statistical analysis
+availability:
+  free: full
+  selfServe: full
+  enterprise: full
+---
+
+import {ProductScreenshot} from 'components/ProductScreenshot'
+
+export const confidenceLight = "https://res.cloudinary.com/dmukukwp6/image/upload/ci_light_964273850b.png"
+export const confidenceDark = "https://res.cloudinary.com/dmukukwp6/image/upload/ci_dark_70b3c2973c.png"
+export const confidenceSettingsLight = "https://res.cloudinary.com/dmukukwp6/image/upload/ci_options_light_faeb378f6c.png"
+export const confidenceSettingsDark = "https://res.cloudinary.com/dmukukwp6/image/upload/ci_options_dark_096f6b4c4b.png"
+
+When analyzing data, it's important to understand if your results are statistically significant. In other words, are the trends you're seeing real, or are they just due to random chance?
+
+PostHog helps you answer this question by providing confidence intervals for your trend lines.
+
+## What are confidence intervals?
+
+A confidence interval is a range of values that likely contains the true value of a metric. A 95% confidence level means that if you were to run the same analysis again 100 times, 95 of those times the true value would fall within the calculated confidence interval.
+
+In PostHog, confidence intervals are displayed as a shaded area around your trend lines. The wider the shaded area, the less confident we are in the data. The narrower the area, the more confident we are.
+
+<ProductScreenshot
+    imageLight={confidenceLight}
+    imageDark={confidenceDark}
+    alt="Confidence intervals on a line chart"
+/>
+
+This is especially useful when you're working with smaller sample sizes (sampling turned on) or when you're trying to detect small changes in your metrics. It helps you avoid making decisions based on noise.
+
+## How to use confidence intervals
```

Comment:
@anirudhpillai Before you ship this, I would actually take this screenshot with a wider aspect ratio, including more of the page its on. It's scrolling on my laptop for ~1.5x viewports, which is a little too much.

Reply:
Thanks so much for the review! This was very helpful 💯